react-delegationdashboard/src/pages/Dashboard/Actions/DelegateAction/DelegateModal.tsx

```diff
@@ -34,7 +34,7 @@ const DelegateModal = ({ show, balance, handleClose, handleContinue }: DelegateM
         denomination,
         decimals,
         showLastNonZeroDecimal: false,
-      }) === contractOverview.maxDelegationCap
+      }) >= contractOverview.maxDelegationCap && contractOverview.maxDelegationCap !== String(0)
```

Comment:
bnAmount.comparedTo(bnAvailable) <= 0; is better (or if you are sure numbers are small, it's OK)